test/notifier.test.js

```diff
@@ -1544,6 +1544,26 @@ describe("Notifier._buildPayload()", function() {
 
     done();
   });
+
+  it("uses raw parsing when any other strategy works", function() {
```

Comment:
Looks like this test is failing in Travis: https://travis-ci.org/rollbar/rollbar.js/builds/51069044